File: code-generator/src/main/java/org/xi/quick/codegenerator/mapper/StatisticsMapper.java
package org.xi.quick.codegenerator.mapper;

import org.apache.ibatis.annotations.Mapper;
import org.apache.ibatis.annotations.Param;
import org.xi.quick.codegenerator.entity.Statistics;

import java.util.List;

@Mapper
public interface StatisticsMapper {

    /**
     * 获取表的索引
     *
     * @param databaseName
     * @param tableName
     * @return
     */
    List<Statistics> getStatistics(@Param("databaseName") String databaseName, @Param("tableName") String tableName);
}

File: mysql-code-generator-kt/src/main/kotlin/org/xi/quick/codegeneratorkt/mapper/StatisticsMapper.kt
package org.xi.quick.codegeneratorkt.mapper

import org.apache.ibatis.annotations.Mapper
import org.apache.ibatis.annotations.Param
import org.xi.quick.codegeneratorkt.entity.Statistics

@Mapper
interface StatisticsMapper {

    /**
     * 获取表的索引
     *
     * @param databaseName
     * @param tableName
     * @return
     */
    fun getStatistics(@Param("databaseName") databaseName: String, @Param("tableName") tableName: String): List<Statistics>
}
